themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -513,86 +499,6 @@ expect(await page.title()).toBe("Pulumi Challenge");
 await browser.close();
 ```
 
-#### Step 7. Introducing the Dynamic Swag Provider
-
-Everyone likes SWAG and we want to give you some for completing this challenge. To do so, we're going to handle this via Pulumi with a Dynamic Provider. Create a new directory and file at `pulumi-challenge/swag-provider/index.ts`:
-
-For this dynamic provider, we can only use CommonJS modules. For making an HTTP request, we can use `got` version `11.8.0`:
-
-```shell
-npm install got@11.8.0
-```
-
-```typescript
-import * as pulumi from "@pulumi/pulumi";
-
-const submittionUrl: string =
-  "https://hooks.airtable.com/workflows/v1/genericWebhook/apptZjyaJx5J2BVri/wflmg3riOP6fPjCII/wtr3RoDcz3mTizw3C";
-
-interface SwagInputs {
-  name: string;
-  email: string;
-  address: string;
-  size: "XS" | "S" | "M" | "L" | "XL" | "XXL" | "XXXL";
-}
-
-interface SwagCreateResponse {
-  success: boolean;
-}
-
-interface SwagOutputs extends SwagInputs {
-  id: string;
-}
-
-class SwagProvider implements pulumi.dynamic.ResourceProvider {
-  private name: string;
-
-  constructor(name: string) {
-    this.name = name;
-  }
-
-  async create(props: SwagInputs): Promise<pulumi.dynamic.CreateResult> {
-    const got = (await import("got")).default;
-
-    let data = await got
-      .post(submittionUrl, {
-        headers: {
-          "Content-Type": "application/json",
-        },
-        json: {
-          ...props,
-        },
-      })
-      .json<SwagCreateResponse>();
-
-    return { id: props.email, outs: props };
-  }
-}
-
-export class Swag extends pulumi.dynamic.Resource {
-  constructor(
-    name: string,
-    props: SwagInputs,
-    opts?: pulumi.CustomResourceOptions
-  ) {
-    super(new SwagProvider(name), name, props, opts);
-  }
-}
-```
-
-Now, add this final block to `pulumi-challenge/index.ts` and run `pulumi up`. Enjoy your SWAG!
-
-```typescript
-import { Swag } from "./swag-provider";
-
-const swag = new Swag("your-startup", {
-  name: "YOUR NAME",
-  email: "YOUR EMAIL",
-  address: "YOUR ADDRESS",
-  size: SIZE,
-});
-```
-
-Congratulations! You completed the first Pulumi Challenge. If you'd like to tear down all of these resources, run `pulumi destroy`. Otherwise, enjoy the new website! Change it around and make it your own. Your swag will be in the mail shortly!
+Congratulations! You completed the first Pulumi Challenge. If you’d like to tear down all of these resources, run pulumi destroy. Otherwise, enjoy the new website!
```

Comment:
Need to add backticks around `pulumi destroy` so that it's shown as a command to be entered